src/node_crypto.cc

```diff
@@ -3148,6 +3148,14 @@ bool CipherBase::InitAuthenticated(const char *cipher_type, int iv_len,
       return false;
     }
 
+#ifdef NODE_FIPS_MODE
+    // TODO(tniessen) Support CCM decryption in FIPS mode
+    if (kind_ == kDecipher && FIPS_mode()) {
+      env()->ThrowError("CCM decryption not supported in FIPS mode");
```

Comment:
The error should be thrown in javascript if at all possible to make use of the `internal/errors` mechanism.

Reply:
Pretty much nothing in node_crypto.cc does that at the moment so I don't think it's reasonable to expect that of this PR.

Reply:
We have been in the process of actively migrating these for quite some time. Also, I said, "if at all possible".

Reply:
Clicking 'Request changes' makes it less than optional though.